Dockerfile.ci

```diff
@@ -6,6 +6,9 @@ ENV BUNDLE_PATH="/home/dependabot/.bundle" \
   BUNDLE_BIN=".bundle/binstubs" \
   PATH=".bundle/binstubs:$PATH:/home/dependabot/.bundle/bin"
 
+# We pre-install these versions of python as they're known to be used in the
+# test suite. These used to be installed in-line, and pre-installing them helps
+# with the performance of the python test suite.
```

Comment:
Where did you find these versions? Could be useful with a ref if we end up wanting to change. Not related 
 to this PR but wondering if we could reduce this to one non-default version?

Reply:
I wrote some code to figure it out which I referenced in the previous PR: https://github.com/dependabot/dependabot-core/commit/9f7251b423dccd474077da9227c3dbfb14f3535f

It's a little gnarly and building a general-purpose version of it would require some cleanup in the python codebase.

Agree it'd be nice to reduce that to a single version, but unsure how much work it would be, and that cutting down the time to install them on CI seemed like a good first step

Reply:
Ah gotcha, maybe something we can hook into notifications that we set up for recording the bundler version?

Wonder where those versions even get extracted from, can't find them in the codebase by grepping 🤔